packages/dashboard-message-bus/README.md

```diff
@@ -30,10 +30,10 @@ The message bus works by running two separate WebSocket servers. One server is f
 
 ### Testing
 
-This package is meant to be used with the `@truffle/dashboard` package. So manual testing is most likely done through the dashboard package. Some automated tests are defined in the `test/` folder, but these still need to be expanded. The message bus functionality is also tested indirectly by the automated tests in the `@truffle/dashboard-provider` and `@truffle/dashboard` packages.
+This package is meant to be used with the `@truffle/dashboard` package. So manual testing is most likely done through the dashboard package. Some automated tests are defined in the `test/` folder, but these still need to be expanded.
 
 ### Adding new message types
 
-Right now there are very few message types. The most important one is `"provider"`, which sends RPC requests for the dashboard-provider. Other message types are `"invalidate"`, which can be sent to invalidate earlier messages, and `"log"`, which is sent by the message bus to send log messages over the wire. The interfaces of these messages are defined inside the `@truffle/dashboard-message-bus` package, from where they can be imported by consumers.
+Right now there are very few message types. The most important one is `"provider"`, which sends RPC requests to the `@truffle/dashboard` so that they can be handled by the injected web3 provider that your wallet (e.g. MetaMask) provides. Other message types are `"invalidate"`, which can be sent to invalidate earlier messages, and `"log"`, which is sent by the message bus to send log messages over the wire. The interfaces of these messages are defined inside the `@truffle/dashboard-message-bus` package, from where they can be imported by consumers.
```

Comment:
Not for this PR, but we should document the message types, or add the current links that do.